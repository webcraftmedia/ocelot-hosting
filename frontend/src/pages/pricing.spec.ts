
import { renderSuspended } from '@nuxt/test-utils/runtime'
import Page from './pricing.vue'
import { describe, expect, it } from 'vitest'

describe('Page: Pricing', () => {
  it('renders', async () => {
    const html = await (await renderSuspended(Page, { route: '/' })).html()
    expect(html).toMatchSnapshot()
  })
})