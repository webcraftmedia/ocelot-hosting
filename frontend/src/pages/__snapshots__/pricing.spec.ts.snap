// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Page: Pricing > renders 1`] = `
"<div id="test-wrapper">
  <div>
    <!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
    <section> Pricing Table: - Test Free - Trial - 14days for free - Up to 100 Users -&gt; 100€/Month - Up to 1000 Users -&gt; 225€/Month - More -&gt; Get in touch </section><!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
    <section>Additonal Services: - Customization: Logo &amp; Colors - Development of Features</section><!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
    <section>Book Trial: - Form</section>
  </div>
</div>"
`;
